### PR TITLE
Task#9753 create ui home screen

### PR DESCRIPTION
**Pull request by anhmv-1205**
titile: create ui home screen

redmi: https://edu-redmine.sun-asterisk.vn/issues/9753

hinh anh
S90405-112000
S90405-111952

### CODE REVIEW

**Comment by anhmv-1205** [comment]
ready

**Comment by tuanlvt-1117** [comment]
commented

**Comment by anhmv-1205** [comment]
ready

File: app/src/main/java/com/sun_asterisk/comics_01/screen/home/adapter/ComicAdapter.java [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    }
+
+    @Override
+    public void onBindViewHolder(@NonNull ComicViewHolder comicViewHolder, int i) {
+        comicViewHolder.bindComic(mComics.get(i));
+    }
+
+    @Override
+    public int getItemCount() {
+        return mComics.size();
```
Comment (tuanlvt-1117) [inline app/src/main/java/com/sun_asterisk/comics_01/screen/home/adapter/ComicAdapter.java]: check null mComics

**Comment by anhmv-1205** [comment]
ready

**Review by tuanlvt-1117 — APPROVED** [review]

**Pull request merged**